### PR TITLE
vcs 1.13.2 (new formula)

### PR DESCRIPTION
**Pull request by RSully**
- [x] Have you followed the guidelines in our [Contributing](https://github.com/Homebrew/homebrew-core/blob/master/.github/CONTRIBUTING.md) document?
- [x] Have you checked to ensure there aren't other open [Pull Requests](https://github.com/Homebrew/homebrew-core/pulls) for the same formula update/change?
- [x] Does your submission pass `brew audit --strict --online <formula>` (where `<formula>` is the name of the formula you're submitting)?
- [x] Have you built your formula locally prior to submission with `brew install <formula>`?

### CODE REVIEW

File: Formula/vcs.rb [hunk 1]
```diff
@@ -0,0 +1,29 @@
+class Vcs < Formula
+  desc "Creates video contact sheets (previews) of videos."
+  homepage "https://p.outlyer.net/vcs/"
+  url "https://p.outlyer.net/vcs/files/vcs-1.13.2.tar.gz"
+  sha256 "fc2a2b3994d5ffb5d87fb3dceaa5f6855aca7a89c58533b12fd11b8fb5b623af"
+
+  depends_on "ffmpeg"
+  depends_on "mplayer" => :optional
+  depends_on "gnu-getopt"
```
Comment (DomT4) [inline Formula/vcs.rb]: Is upstream aware they are lacking compatibility with BSD `getopt`?

File: Formula/vcs.rb [hunk 2]
```diff
@@ -0,0 +1,29 @@
...
+    inreplace "vcs" do |s|
+      s.gsub! "declare GETOPT=getopt", "declare GETOPT=#{Formula["gnu-getopt"].opt_bin}/getopt"
+    end
+
+    system "make", "install", "prefix=#{prefix}"
+  end
+
+  test do
+    system "#{Formula["ffmpeg"].opt_bin}/ffmpeg", "-y", "-filter_complex",
+      "testsrc=rate=1:duration=2", "#{testpath}/video.mp4"
```
Comment (DomT4) [inline Formula/vcs.rb]: Style nit but it's a bit easier to work out this is part of the same argument set if you align the first `"` with the first `"` on the line above.

File: Formula/vcs.rb (hunk 1 above)
Comment (RSully) [inline Formula/vcs.rb]: They specifically require "enhanced" `getopt`, which they check for using `getopt -T`.

File: Formula/vcs.rb (hunk 2 above)
Comment (RSully) [inline Formula/vcs.rb]: Fixed in latest push.

File: Formula/vcs.rb (hunk 1 above)
Comment (mikemcquaid) [inline Formula/vcs.rb]: At build or runtime or both?

File: Formula/vcs.rb [hunk 3]
```diff
@@ -0,0 +1,29 @@
...
+
+  depends_on "ffmpeg"
+  depends_on "mplayer" => :optional
+  depends_on "gnu-getopt"
+  depends_on "ghostscript"
+  depends_on "imagemagick"
+
+  def install
+    inreplace "vcs" do |s|
+      s.gsub! "declare GETOPT=getopt", "declare GETOPT=#{Formula["gnu-getopt"].opt_bin}/getopt"
```
Comment (mikemcquaid) [inline Formula/vcs.rb]: You can use the `inreplace` three argument form as you're just making a single replacement.

File: Formula/vcs.rb (hunk 1 above)
Comment (RSully) [inline Formula/vcs.rb]: Runtime

**Comment by DomT4** [comment]
Merged in https://github.com/Homebrew/homebrew-core/commit/dc3265fe1e89c8e336af059c1b003bc894cd5627. Thank you for your contribution to Homebrew @RSully; we appreciate it! :smiley_cat:

**Pull request closed without merging**